src/Microsoft.DotNet.Web.Spa.ProjectTemplates/content/ReactRedux-CSharp/ClientApp/src/index.tsx

```diff
@@ -1,29 +1,29 @@
 import 'bootstrap/dist/css/bootstrap.css';
-import React from 'react';
-import ReactDOM from 'react-dom';
+
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
 import { Provider } from 'react-redux';
-import { ConnectedRouter } from 'react-router-redux';
+import { ConnectedRouter } from 'connected-react-router';
 import { createBrowserHistory } from 'history';
 import configureStore from './store/configureStore';
+import { ApplicationState } from './store';
 import App from './App';
 import registerServiceWorker from './registerServiceWorker';
 
 // Create browser history to use in the Redux store
-const baseUrl = document.getElementsByTagName('base')[0].getAttribute('href');
+const baseUrl = document.getElementsByTagName('base')[0].getAttribute('href') as string;
 const history = createBrowserHistory({ basename: baseUrl });
 
 // Get the application-wide store instance, prepopulating with state from the server where available.
-const initialState = window.initialReduxState;
+const initialState = (window as any).initialReduxState as ApplicationState;
```

Comment:
I don't think *prepopulating with state from the server where available* is applicable any more, since the server-side code no longer supports SSR since we moved to CRA. The `initialState` could be eliminated from the code here.

Reply:
Fixed